moped-editor/src/views/projects/projectView/MilestoneTemplateModal.js

```diff
@@ -38,12 +41,12 @@ const templateChoices = ["AMD", "PDD"];
  * useMemo hook to choose milestone options
  * @return {Object[]}
  */
-const useMilestoneOptions = (template, projectId, selectedMilestonesIds) =>
+const useMilestoneOptions = (template, projectId) =>
   useMemo(() => {
     if (template === "AMD") {
       return returnSignalPHBMilestoneTemplate(projectId);
-      // } else if (template === "PDD") {
-      //   // etc return [];
+    } else if (template === "PDD") {
```

Comment:
@chiaberry would you mind changing the template names to `Arterial Management` and `Project Deliver` instead of AMD and PPD? Channeling my inner @amenity :)